backend/akr/src/test/java/fi/oph/akr/scheduled/ExpiringAuthorisationsEmailCreatorTest.java

```diff
@@ -85,17 +86,38 @@ public void testCheckExpiringAuthorisations() {
     createAuthorisationTermReminder(remindedAuth2);
     createAuthorisationTermReminder(remindedAuth2);
 
-    emailCreator.checkExpiringAuthorisations();
+    final Translator t9 = createTranslator("t9@invalid");
+    final Authorisation qExpiringButHasEquivalentAuthorisationNotExpiring = createAuthorisation(t9, meetingDate, date);
+    final Authorisation qNotExpiringWhichPreventsEquivalentAuthorisationReminder = createAuthorisation(
+      t9,
+      meetingDate,
+      date.plusMonths(3).plusDays(1)
+    );
 
-    verify(clerkEmailService, times(3)).createAuthorisationExpiryEmail(longCaptor.capture());
+    final Translator t10 = createTranslator("t10@invalid");
+    final Authorisation qExpiringButHasEquivalentAuthorisationAlsoExpiring = createAuthorisation(
```

Comment:
qExpiringButHasEquivalentAuthorisationExpiringLater ?

Reply:
Nimennät täällä musta ok. Kävi tosin lähinnä sellainen mielessä, että vaihtoehtoisesti nämä `t10`:lle määritellyt auktorisoinnit voisi sen sijaan olla `t9`:n alaisia auktorisointeja esim. eri kieliparille (`from` tai `to` jotain muuta). Sitä kautta tulisi sitä filteröintiäkin testattua ainakin kieliparin mukaan.

Reply:
Hyvä idea.